### DISCUSSION TITLE
How do I display the email address on the login-update-password.ftl page?

### DISCUSSION THREAD

**Discussion by huangjy6688** [keycloak/keycloak#22663] [Q&A]
### Description

How do I display the email address on the login-update-password.ftl page?
When email is not used as username,

<input type="text" id="username" name="username" value="${username}" autocomplete="username"
readonly="readonly" style="display:none;" />

email cannot be displayed using ${email} or ${user.email}.

**Comment by huangjy6688** [comment]
Who will make it happen? Thank you

**Reply by codespearhead** [reply]
@huangjy6688 this is technically a duplicate of #28282 . Can you close this discussion in favor of that one? There's #29044 too since we're still working on centralizing related problems.
